assets/pages/digital-subscription-checkout/components/checkoutForm.jsx

```diff
@@ -200,6 +202,9 @@ function CheckoutForm(props: PropTypes) {
         />
         {errorState}
         <Button1 onClick={() => props.submitForm()}>Continue to payment</Button1>
+        <DirectDebitPopUpForm
+          onPaymentAuthorisation={(pr) => { props.onPaymentAuthorised(pr); }}
```

Comment:
Minor: on first glance I'm not sure what _pr_ is. It might be worth renaming this and/or adding a type?